proto/tendermint/p2p/types.proto

```diff
@@ -29,10 +29,12 @@ message NodeInfoOther {
 }
 
 message PeerInfo {
-  string                    id             = 1 [(gogoproto.customname) = "ID"];
-  repeated PeerAddressInfo  address_info   = 2;
-  google.protobuf.Timestamp last_connected = 3 [(gogoproto.stdtime) = true];
-  bool                      inactive       = 4;
+  string                    id                = 1 [(gogoproto.customname) = "ID"];
+  repeated PeerAddressInfo  address_info      = 2;
+  google.protobuf.Timestamp last_connected    = 3 [(gogoproto.stdtime) = true];
+  bool                      inactive          = 4;
+  google.protobuf.Timestamp last_disconnected = 5 [(gogoproto.stdtime) = true];
```

Comment:
The cooldown is going to be order of seconds/milliseconds, I think it's not that meaningful to persist it.

Reply:
Connection throttle is a few ms. You'd have to crash and recover in an instant for that to be hit?

Reply:
ok ok